### ISSUE TITLE
Interact Key UI on logs appears on center of trunk

### ISSUE THREAD

**Issue by Nanders** [StrangeLoopGames/EcoIssues#3276]
Rather than where you are targeting, making it often hard to see the prompt

![image](https://user-images.githubusercontent.com/6262198/34699412-08df9da0-f492-11e7-90ec-a54d4eacb601.png)

**Comment by MarjieAVolk** [comment]
This might be https://github.com/StrangeLoopGames/EcoIssues/issues/3269

**Comment by Nanders** [comment]
I don't think so, the transform of the trunk is at the center, and is appropriately where the marker is. 

I just don't think it should be strictly limited to the origin of the object you're looking at.
